content/developer/packaging/basics.zh.md

```diff
@@ -163,74 +165,84 @@ PKGCONFL="i3-gaps"
 
 该文件夹存放构建前要应用的补丁文件，将补丁文件放入该文件夹即可。
 
-# 实操案例：`light`
+# 实操案例：`Hello`
 
-接下来，我们来实战软件包配置编写。该章节介绍 [light](https://github.com/haikarainen/light) 这个简单软件包的构建配置编写方式。该软件提供一个用于控制笔记本背光的简单命令，而且因为实现原理只依赖文件 API，该软件只依赖 `glibc`。
+接下来，我们来实战软件包配置编写。该章节介绍 [hello](https://www.gnu.org/software/hello/) ，这是一个由 GNU 编写的简易问候程序，可在屏幕上打印一句“Hello, world!”。本软件没有依赖项，故而非常适合入门。
 
-首先切换到 `TREE` 目录，并确定我们目前处于正确的 Git 分支。如前面提到的主题制迭代流程所要求，您需要首先为这个包创建一个 Git 分支。由于要引入新软件包，根据 [AOSC OS 主题制维护指南 (AOSC OS Topic-Based Maintenance Guidelines)](@/developer/packaging/topic-based-maintenance-guideline.md)，分支名称应为 `$PKGNAME-$PKGVER-new`，即 `light-1.2.1-new`。
+首先切换到 `TREE` 目录，并确定我们目前处于正确的 Git 分支。如前面提到的主题制迭代流程所要求，您需要首先为这个包创建一个 Git 分支。由于要引入新软件包，根据 [AOSC OS 主题制维护指南 (AOSC OS Topic-Based Maintenance Guidelines)](@/developer/packaging/topic-based-maintenance-guideline.md)，分支名称应为 `$PKGNAME-$PKGVER-new`，即 `hello-2.12-new`。
 
-因为 `light` 属于实用工具，我们要在 `extra-utils` 下创建 `light` 目录。
+因为 `hello` 属于实用工具，我们要在 `app-utils` 下创建 `hello` 目录。
 
 ``` bash
-cd TREE/extra-utils
-mkdir light
-cd light
+cd TREE/app-utils
+mkdir hello
+cd hello
 ```
 
-接下来，编写 `spec` 文件。首先，查阅项目网站并辨认最新版本源码包的下载 URL。在计算 `sha256` 校验和后，我们即可将各项信息填入文件。
+接下来，编写 `spec` 文件。首先，查阅项目网站，并找到最新版本源码包的下载 URL；下载文件并确认无误后，需计算其 `SHA-256` 哈希值（可使用 `sha256sum` 命令），并将各项信息填入文件。
 
 ``` bash
-VER=1.2.1
-SRCS="tbl::https://github.com/haikarainen/light/archive/v$VER.tar.gz"
-CHKSUMS="sha256::53d1e74f38813de2068e26a28dc7054aab66d1adfedb8d9200f73a57c73e7293"
+VER=2.12.1
+SRCS="tbl::https://ftp.gnu.org/gnu/hello/hello-$VER.tar.gz"
+CHKSUMS="sha256::8d99142afd92576f30b0cd7cb42a8dc6809998bc5d607d88761f512e26c7db20"
```

Comment:
Maybe explain how to get this (i.e. `sha256sum hello-2.12.tar.gz`) and mention other supported hash algorithms.